ocpmodels/datasets/trajectory_lmdb.py

```diff
@@ -29,13 +29,15 @@ def __init__(self, config):
         ]
 
         self.inds = []
+        self.transform = None
```

Comment:
Can you add this to the `__init__`

`def __init__(self, config, transform=None)`
.
.
.
`self.transform = None`

Reply:
Fixed!